components/camera/videosource/helpers.go

```diff
@@ -1,125 +0,0 @@
-package videosource
-
-import (
-	"bytes"
-	"context"
-	"image"
-	"io"
-	"net/http"
-	"strings"
-
-	"github.com/pkg/errors"
-	viamutils "go.viam.com/utils"
-
-	"go.viam.com/rdk/gostream"
-	"go.viam.com/rdk/logging"
-	"go.viam.com/rdk/rimage"
-	"go.viam.com/rdk/utils"
-)
-
-// getMIMETypeFromData uses context to determine a MIME type requested by a parent
-// process and attempts to detect the MIME type of the data from its header.
-// If no MIME type has been requested or there is a mismatch, it returns the one
-// detected by http.DetectContentType.
-func getMIMETypeFromData(ctx context.Context, data []byte, logger logging.Logger) (string, error) {
```

Comment:
I guess this function is not used anywhere else?

Reply:
<img width="319" alt="Screenshot 2024-04-30 at 4 08 07 PM" src="https://github.com/viamrobotics/rdk/assets/55464069/034fa4f0-2369-4ed0-be3a-b20edfa7aa95">

Yep confirmed above and linter was yellow underlining all the functions in that file as unused.